libs/import/src/lib/utils.ts

```diff
@@ -7,7 +7,11 @@ import { SheetTab, ValueWithError } from "@blockframes/utils/spreadsheet";
 import { centralOrgId } from "@env";
 import { ContractService } from "@blockframes/contract/contract/+state/contract.service";
 import { UserService } from "@blockframes/user/+state";
+import { CollectionService } from "akita-ng-fire";
 
+export abstract class CheckDocExists {
+  abstract docExists(docId: string);
+}
```

Comment:
If you don't have any methods or properties, use an interface instead :
```suggestion
export interface CheckDocExists {
  docExists(docId: string): Promise<boolean>;
}
```